src/URDFConverter/UrdfUnity/UrdfUnity/FileManager/FileManager.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.IO;
+using UrdfUnity.Urdf.Models;
+using UrdfUnity.Util;
+using UrdfUnity.Parse;
+using NLog;
+
+
+namespace UrdfUnity.FileManager
+{
+    /// <summary>
+    /// Filetypes we care about in the file manager.
+    /// </summary>
+    public enum FileType
+    {
+        UNKNOWN,
+        URDF,
+        XACRO
+    }
+
+
+    /// <summary>
+    /// This class will be handling the FileIO 
+    /// </summary>
+    public class FileManager
+    {
+        Logger LOGGER;
+        
+
+        /// <summary>
+        /// Base constructor, for initializing components
+        /// </summary>
+        public FileManager()
+        {
+            // initializing the logger for this class.
+            LOGGER = LogManager.GetCurrentClassLogger();
+        }
+
+
+        /// <summary>
+        /// We have a URDF/XACRO filepath that needs to be extracted to be
```

Comment:
Nitpick again.  Method comments generally begin with a verb: "Extracts a Robot model object from the provided URDF/Xacro filepath."